.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-package-lock.json
\ No newline at end of file
+./package-lock.json
```

Comment:
Curious about this change. Is this because you will have multiple package-lock.json files?

Reply:
No, it just kept showing up as being changed when I would do a 'git status'